docs/app/Examples/elements/Segment/Groups/SegmentHorizontalSegmentsExample.js

```diff
@@ -4,11 +4,11 @@ import { Segment } from 'stardust'
 export default class SegmentHorizontalSegmentsExample extends Component {
   render() {
     return (
-      <Segment.Segments className='horizontal'>
+      <Segment.Group aligned='horizontal'>
```

Comment:
Since the [`horizontal`](http://semantic-ui.com/elements/segment.html#horizontal-segments) class here applies to the layout of the segments and there is only one option, I think we should use a boolean prop:

```js
<Segment.Group horizontal />
```

![image](https://cloud.githubusercontent.com/assets/5067638/16365125/b53c0eae-3bab-11e6-8a78-00810e7c74bb.png)


Please let me know if I'm over looking anything.